catalog/core/catalog-core-versioning/versioning-common/src/main/java/ddf/catalog/core/versioning/impl/MetacardVersionImpl.java

```diff
@@ -215,7 +215,7 @@ public static Metacard toMetacard(Metacard source, List<MetacardType> types) {
    * @return The original metacard this version represents
    */
   public static Metacard toMetacard(Metacard source) {
-    String id = (String) source.getAttribute(MetacardVersion.VERSION_OF_ID).getValue();
+    String id = (String) source.getAttribute("id").getValue();
```

Comment:
This definitely seems like a breaking change if we are no longer setting the `VERSION_OF_ID` on the metacard